### DISCUSSION TITLE
Getting this error while adding image generation url

### DISCUSSION THREAD

**Discussion by Yerli-Oyuncu** [open-webui/open-webui#2356] [Q&A]
![image](https://github.com/open-webui/open-webui/assets/62718300/7e61f7c0-a390-4d1c-bff9-fa1e6c07475c)

**Comment by spammenotinoz** [comment]
Suggest trying with the actual IP of the host, ensure you started automatic with the --api but not --api-auth
127.0.0.1 is localhost likely referring to the docker container.
